br/backup-and-restore-tool.md

```diff
@@ -159,20 +153,11 @@ Currently, the following methods are supported to run the BR tool:
 
 #### Use SQL statements
 
-In TiDB v4.0.2 and later versions, you can run the BR tool using SQL statements.
-
-For detailed operations, see the following documents:
-
-- [Backup syntax](/sql-statements/sql-statement-backup.md#backup)
-- [Restore syntax](/sql-statements/sql-statement-restore.md#restore)
+TiDB supports both [`BACKUP`](/sql-statements/sql-statement-backup.md#backup) and [`RESTORE`](/sql-statements/sql-statement-restore.md#restore) SQL statements. The progress of these operations can be monitored with the statement [`SHOW BACKUPS|RESTORES`](/sql-statements/sql-statement-show-backups.md).
 
 #### Use the command-line tool
 
-In TiDB versions above v3.1, you can run the BR tool using the command-line tool.
-
-First, you need to download the binary file of the BR tool. See [download link](/download-ecosystem-tools.md#br-backup-and-restore).
-
-For how to use the command-line tool to perform backup and restore operations, see [Use the BR command-line tool](/br/use-br-command-line-tool.md).
+The `br` command-line utility is available as a separate download. For details, see [Use BR Command-line for Backup and Restoration](/br/use-br-command-line-tool.md).
```

Comment:
[Use BR Command-line for Backup and Restoration](/br/use-br-command-line-tool.md) does not provide the download link?